app/views/service_requests/review.html.haml

```diff
@@ -19,7 +19,7 @@
 -# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 - if SYSTEM_SATISFACTION_SURVEY != false
-  = javascript_include_tag 'review'
```

Comment:
Does this page have a 'Save As Draft' option?  If so, it needs the 'review' code as well so that it give the appropriate messages